src/main/java/com/uwetrottmann/tmdb2/exceptions/TmdbAuthenticaitonFailedException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Nikolas Mavropoylos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.uwetrottmann.tmdb2.exceptions;
+
+public class TmdbAuthenticaitonFailedException extends TmdbException {
```

Comment:
Yeah it is fixed, just not commited yet, will do with the tests, i don't want many commits for no reason at all